priv/templates/alert.eex

```diff
@@ -0,0 +1,102 @@
+defmodule <%= @module_name %>Web.Components.Alert do
+  @moduledoc false
+  use SaladUI, :component
```

Comment:
I think we should setup a macro, so user can add `salad_ui` for dev environment only

Reply:
I’m not sure which macro you're referring to or how it should behave. Could you clarify that for me?

Reply:
We can do something like this

```elixir
defmodule <module_prefix>.Component
defmacro __using__(which) when is_atom(which) do
    quote do
      use Phoenix.Component

      import <module_prefix>.Helpers
      import Tails, only: [classes: 1]

      alias Phoenix.LiveView.JS
    end
  end
end
```

Task Salad.init should copy to target directory
and for each component, replace 
`use SaladUI, :component`
by
`use <module_prefix>.Component`

This way when the project is compiled, it won't compile `salad_ui` because user only need it for dev environment